common/saber_base.h

```diff
@@ -38,6 +38,13 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(SECONDARY_IGNITION)		\
+    DEFINE_EFFECT(SECONDARY_RETRACTION)		\
+    DEFINE_EFFECT(QUOTE)			\
+    /* Change Alt font sounds */                \
+    DEFINE_EFFECT(ALT_SOUND)			\
+    /* Transition Effect/Pulse Sound */         \
```

Comment:
This comment is also pretty unhelpful.
Transition from what to what?
What would trigger this effect?

Reply:
updated the comment, what triggers this effect depends entirely on the style itself, this is part of the new transition effect / pulse capabilities all tied to the alt font and tr sound parts we discussed.